modules/python/pylib/Makefile.am

```diff
@@ -66,13 +66,8 @@ clean-pylib:
 
 python-checks: python-unit python-pep8 python-pylint
 
-if NOSETESTS
 python-unit:
```

Comment:
can you please merge python-* targets with pytest-* targets? I don't see we need both, and since the prefix "python-" seems more general, I think pytest-* should just be shoved inside the python targets (and probably be promoted to the root Makefile instead of being under modules/python/pylib).

The best would be if we had very specific targets under modules/python/pylib/ and tests/python_functional/ which would be depended on by a generic target in the root Makefile.

Reply:
Thanks for the comment, I will think over it.

Reply:
If I understand you correctly, something like this would be the result:
root/Makefile:
`python-checks` (as a new target): which runs all python-specified targets from `modules/python/pylib/` and from  `tests/python_functional/`

modules/python/pylib/Makefile:
`python-module-checks` (as a new target name for this specified directory): which runs current targets from `modules/python/pylib/`, maybe renaming is needed to not conflicting with targets under `tests/python_functional/ `

tests/python_functional/Makefile:
`python-functional-checks` (as a new target name for this specified directory): which runs current targets from `tests/python_functional/`, maybe with renaming to not conflict with `modules/python/pylib/` targets

If this would be a goal, I am ok with it in this PR (maybe good target names will be hard to choose).

And what about "old" func-test target from: tests/functional/Makefile.am? Should I include it also?

Reply:
Since there was answer to these questions, and we discussed most of them IRL, I mark this thread resolved.